src/components/Newsletter/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import withSizes from 'react-sizes'
+import NewsletterFormDesktop from './NewsletterFormDesktop'
+import NewsletterFormMobile from './NewsletterFormTablet'
+
+const NewsletterForm = props => {
+  if (props.isTablet) {
+    return <NewsletterFormMobile {...props} />
+  }
+
+  return <NewsletterFormDesktop {...props} />
```

Comment:
This kind of responsive layout is really easy to do with media queries + flexbox instead of separate react components + react-sizes. It's also way faster to write (once you've learned it, of course!) and ends up being less code. I'm happy for this to be merged—no point reimplementing something that works!—but if you want to learn more about responsive styling, let me know and I can help you refactor this 👨‍💻👨‍💻